LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/LoRaDeviceFactory.cs

```diff
@@ -41,7 +41,7 @@ public Task<LoRaDevice> CreateAndRegisterAsync(IoTHubDeviceInfo deviceInfo, Canc
         {
             _ = deviceInfo ?? throw new ArgumentNullException(nameof(deviceInfo));
 
-            if (string.IsNullOrEmpty(deviceInfo.PrimaryKey) || string.IsNullOrEmpty(deviceInfo.DevEUI))
+            if (string.IsNullOrEmpty(deviceInfo.PrimaryKey) || deviceInfo.DevEUI == default)
```

Comment:
Better to check for validity:

```suggestion
            if (string.IsNullOrEmpty(deviceInfo.PrimaryKey) || !deviceInfo.DevEUI.IsValid)
```